src/app/core/admin/admin-entity-details/admin-entity-details/admin-entity-details.component.html

```diff
@@ -19,32 +19,59 @@
     let-tabIndex="index"
   >
     <div class="flex-column gap-large padding-top-large">
-      @for (
-        componentConfig of item.components;
-        track componentConfig;
-        let i = $index
-      ) {
-        <div class="section-wrapper">
-          <app-admin-section-header
-            [(title)]="componentConfig.title"
-            (remove)="item.components.splice(i, 1)"
-          ></app-admin-section-header>
-          @if (componentConfig.component === "Form") {
-            <app-admin-entity-form
-              [config]="componentConfig.config"
-              (configChange)="componentConfig.config = $event"
-              [entityType]="entityConstructor"
-              uniqueAreaId="{{ tabIndex }}-{{ i }}"
-            >
-            </app-admin-entity-form>
-          } @else {
-            <app-admin-entity-panel-component
-              [config]="componentConfig"
-              [entityType]="entityConstructor"
-            ></app-admin-entity-panel-component>
-          }
-        </div>
-      }
+      <mat-accordion
+        cdkDropList
+        [cdkDropListData]="item.components"
+        cdkDropListOrientation="vertical"
+        (cdkDropListDropped)="onSectionDrop(item, $event)"
+      >
+        @for (
+          componentConfig of item.components;
+          track componentConfig;
+          let i = $index
+        ) {
+          <mat-expansion-panel
+            class="section-wrapper"
+            [expanded]="isSectionExpanded(item, componentConfig)"
```

Comment:
when testing with a breakpoint, it shows that the `isSectionExpanded()` method gets called endlessly after switching a tab ... Because it is a method call in the template, every change detection cycle will execute it.

We should change this to a property somehow. Maybe a signal that just has an array of opened states that can be accesses by index?